Dockerfile

```diff
@@ -1,10 +1,11 @@
 FROM ubuntu:18.04
```

Comment:
could you maybe use the python docker images? The ubuntu image takes up a lot of space.

Reply:
I just experimented with some alternative images, there are strong advices not to user alpine, there are issues with some packages(libmysqlclient-dev) for which i couldn't find a quick fix at the moment.

Reply:
I just thought maybe use the python docker image
https://hub.docker.com/_/python?tab=description